crates/runtime-configs/src/lib.rs

```diff
@@ -139,6 +141,7 @@ impl slight_events_api::Watch for ConfigsInner {}
 /// As per its' usage in `ConfigsInner`, it must `derive` `Debug`, and `Clone`.
 #[derive(Debug, Clone)]
 pub enum ConfigsImplementor {
+    Local,
```

Comment:
What are the usages of this Local config implementator?

Reply:
~local is for configs that are defined directly in the `toml` file. Here's an example: 

```toml
specversion = "0.1"
[[capability]]
resource = "kv.azblob"
name = "customers"
  [capability.configs]
  c = "world"
```

Reply:
Can you mix them?
```toml
[capability.configs]
c = "world"
d = "${azapp.d}"
```

Reply:
Yep~ You can mix them. They are all individually parsed.